service/src/main/resources/application.yaml

```diff
@@ -359,7 +359,7 @@ robotics:
     recipient: FamilyPublicLaw+robotics@gmail.com
 
 scheduler:
-  enabled: false
```

Comment:
is this wanted or a typo?

Reply:
good spot, was testing something and this slip in

Reply:
done